x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -50,13 +53,29 @@ const pidValidator = (argData: ParsedArgData): true | string => {
   }
 };
 
+const executeTimeoutValidator = (argData: ParsedArgData): true | string => {
+  const emptyResult = emptyArgumentValidator(argData);
```

Comment:
This is no longer needed. See my other comment below on the command definition

Reply:
997bde19c8da4e7798793f8fcd7041902412a6f1